PatientRegistrationApp/src/components/HouseholdRegisterNumber.vue

```diff
@@ -34,7 +34,7 @@ watch: {
 			const householdCountPayload = {
 				householdCount: this.e1
 			}
-			EventBus.$emit('DATA_HOUSHOLD_COUNT_UPDATED', householdCountPayload)
+			EventBus.$emit('DATA_HOUSEHOLD_COUNT_UPDATED', householdCountPayload)
```

Comment:
Another application state example!

Reply:
It's just a message to send the "number of household members" data to the App.vue page when a value changes.

Reply:
Yep just pointing out like earlier!